django_messages/models.py

```diff
@@ -79,7 +79,7 @@ def __str__(self):
 
     def get_absolute_url(self):
         return ('messages_detail', [self.id])
```

Comment:
shouldn't we change this row to `return reverse('messages_detail', [self.id])` when we remove the decorator?

Reply:
> return reverse('messages_detail', [self.id]

Yep, definitely, we need to return a value with reverse, if not we will get an error

Reply:
> return reverse('messages_detail', [self.id]

Yep, definitely, we need to return a value with reverse, if not we will get an error